main.go

```diff
@@ -325,6 +326,19 @@ func init() {
 		}
 	}
 
+	if config.GCP.CloudRun.Endpoint != "" {
```

Comment:
```suggestion
	if config.GCP.CloudRun.Endpoint != "" && config.GCP.CloudRun.JWT != "" {
```

Reply:
it should work without providing JWT, so I'm not sure  I should do this 🤔